crates/pipeline_manager/src/compiler.rs

```diff
@@ -493,6 +493,7 @@ inherits = "release"
             CompilerConfig::crate_name(program_id),
         );
         let toml_path = config.workspace_toml_path();
+        println!("writing '{}'", toml_path.display());
```

Comment:
debug line?

Reply:
Yeah, I've had to evolve this. I'll force-push it out when it fully works.